README.md

```diff
@@ -9,3 +9,9 @@ docker-compose build
 docker-compose run local bash /src/build-and-test-java.sh
 docker-compose run local bash /src/build-and-test-python.sh
 ```
+
+### Run individual larky stdlib test
+
+```bash
+mvn -Dtest='StdLibTest*' -Dlarky.stdlib_test=test_bytes.star org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M5:test -pl larky
```

Comment:
```suggestion
mvn -Dtest='StdLibTest*' -Dlarky.stdlib_test=test_bytes.star test -pl larky
```

this should work because you've changed the default surefire plugin to 3.0 already, no need to override.